test/navigation/drive_test.py

```diff
@@ -6,7 +6,12 @@
 from util.SO3 import SO3
 from math import pi
 from util.np_utils import angle_to_rotate
+from navigation.gate import GateTrajectory
 from tf.transformations import quaternion_from_euler
+import rospy
+
+
+APPROACH_DISTANCE = 2.0
 
 
 class TestDrive(unittest.TestCase):
```

Comment:
was gate test deleted? I don't see it anymore but its still referenced in the .test files. I think we want that file and make sure that the gate path related tests are in them.